src/lib/detection_algorithms/cntk/test/test_exporter.js

```diff
@@ -0,0 +1,19 @@
+const YoloExporter = require('../main.js').Exporter;
+const path = require('path');
+
+var exporter = new YoloExporter(path.join(__dirname, 'export_test'), ['cat', 'dog', 'mouse'], 1000, 1000);
```

Comment:
I will update it to test the cntk version, since the inputs are the same we can use it as a sanity check